### PR TITLE
Make CIFuzz more debuggable by allowing one to run off a branch

### PR DESCRIPTION
**Pull request by jonathanmetzman**
Make CIFuzz incorporate changes to a branch if one specifies a branch to use CIFuzz from.

### CODE REVIEW

File: infra/run_fuzzers.Dockerfile
Comment (inferno-chromium) [inline infra/run_fuzzers.Dockerfile:27]: Is removal required ?

**Review by inferno-chromium — APPROVED** [review]

**Comment by jonathanmetzman** [comment]
Waiting until Monday to land.

File: infra/run_fuzzers.Dockerfile
Comment (jonathanmetzman) [reply infra/run_fuzzers.Dockerfile:27]: It might not be strictly necessary in most cases since we overwrite the files in the directory. But there are some weird edge cases I can think of where deleting the directory matters and so I'm not sure it's worth it to not do the removal now.

I can do the removal when building cifuzz-base to avoid slowing things down.

**Pull request merged**
